src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,7 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user = new UserService().findByEmail(email);
```

Comment:
Create new user service in field, because you need only one object, every invoke of login method will create one more

Reply:
New UserService field created